src/Sarif/Schemata/sarif-2.1.0-rtm.1.json

```diff
@@ -1,7 +1,7 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
we should update release history!

Reply:
Thanks, will do.

---
In reply to: [291363022](https://github.com/microsoft/sarif-sdk/pull/1523#discussion_r291363022) [](ancestors = 291363022)

Reply:
Done.

---
In reply to: [291365943](https://github.com/microsoft/sarif-sdk/pull/1523#discussion_r291365943) [](ancestors = 291365943,291363022)